lua/lsp-inlayhints/core.lua

```diff
@@ -18,7 +18,16 @@ vim.lsp.handlers["workspace/inlayHint/refresh"] = function(_, _, ctx)
   return vim.NIL
 end
 
-local function set_store(bufnr, client)
+local function set_store(client, bufnr)
+  -- TODO Remove
+   if type(bufnr) == "table" and type(client) == "number" then
+     vim.notify_once(
+       "[LSP Inlayhints] on_attach should be called with (client, bufnr)",
+       vim.log.levels.WARN
+     )
+
+     client, bufnr = bufnr, client
+   end
```

Comment:
We don't need the block here since we've already corrected it before (`set_store` is only called on `on_attach`).

Reply:
done